src/main/java/core/basesyntax/Lottery.java

```diff
@@ -3,9 +3,11 @@
 import java.util.Random;
 
 public class Lottery {
+    public static final Random RANDOM = new Random();
+
     public Ball getRandomBall() {
-        int randomLotteryBall = new Random().nextInt(100);
+        int randomBallNumber = RANDOM.nextInt(100);
         ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
move creating instance of `ColorSupplier` to a class field near `Random`

Reply:
so other methods can get access or just good manners?

Reply:
creating new instance is a time consuming operation in Java. This reduces performance. So good practice is to create a new instance only when it is necessary.